app/BeHost/BeHost.jsx

```diff
@@ -0,0 +1,262 @@
+import Header from "../src/components/Header/Header";
+import Footer from "../src/components/Footer/Footer";
+import { useState } from "react";
+import image from "../src/assets/spices.jpg";
+import styles from "./BeHost.module.css"; // Ensure correct import of CSS module
+import TextField from "@mui/material/TextField";
+import { DateField } from "@mui/x-date-pickers/DateField";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { DemoContainer } from "@mui/x-date-pickers/internals/demo";
+import InputAdornment from "@mui/material/InputAdornment";
+import StyledButton from "../src/components/StyledButton/StyledButton";
+import axios from "axios";
+
+export default function BeHost() {
+  const [price, setPrice] = useState("");
```

Comment:
consider adding the logic of this file into another component(s) and then have this file pretty clean :)

Reply:
Yeah my next step will be actually using context and reducer so things would be more clean , if i understand correct :)